var/spack/repos/builtin/packages/rocm-suite/package.py.save

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Comment:
Echoing @cedricchevalier19's comment above, this file seems to be a backup file that was inadvertently added to the commit.